src/main/java/adris/altoclef/tasks/misc/GetToOuterEndIslandsTask.java

```diff
@@ -0,0 +1,120 @@
+package adris.altoclef.tasks.misc;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.Debug;
+import adris.altoclef.tasks.InteractWithBlockTask;
+import adris.altoclef.tasks.movement.GetToYTask;
+import adris.altoclef.tasks.resources.GetBuildingMaterialsTask;
+import adris.altoclef.tasks.speedrun.BeatMinecraft2Task;
+import adris.altoclef.tasks.squashed.CataloguedResourceTask;
+import adris.altoclef.tasksystem.Task;
+import adris.altoclef.util.Dimension;
+import adris.altoclef.util.ItemTarget;
+import adris.altoclef.util.baritone.GoalAnd;
+import adris.altoclef.util.helpers.StorageHelper;
+import adris.altoclef.util.helpers.WorldHelper;
+
+import baritone.api.pathing.goals.GoalComposite;
+import baritone.api.pathing.goals.GoalGetToBlock;
+import baritone.api.pathing.goals.GoalYLevel;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+import net.minecraft.util.math.Vec3i;
+
+public class GetToOuterEndIslandsTask extends Task {
+    private Task _beatTheGame;
+    private boolean _pathingToGateway = false;
+    public final Vec3i[] OFFSETS = {
+            new Vec3i(1, -1, 1),
+            new Vec3i(1, -1, -1),
+            new Vec3i(-1, -1, 1),
+            new Vec3i(-1, -1, -1),
+            new Vec3i(2, -1, 0),
+            new Vec3i(0, -1, 2),
+            new Vec3i(-2, -1, 0),
+            new Vec3i(0, -1, -2)
+    };
+    public GetToOuterEndIslandsTask() {
+
+    }
+
+    @Override
+    protected void onStart(AltoClef mod) {
+        mod.getBehaviour().push();
+        mod.getBlockTracker().trackBlock(Blocks.END_GATEWAY);
+        _beatTheGame = new BeatMinecraft2Task();
+    }
+
+    @Override
+    protected Task onTick(AltoClef mod) {
+        if (isFinished(mod)) {
```

Comment:
I am not sure you need this here at all. If `this` is finished, it will end `onTick` anyway, so something else is going on if this was the solution to a problem.

Reply:
I would remove this as it may cause unexpected problems

Reply:
I mean sure

Reply:
Wait would this cause a problem with `isActive`?

Reply:
Nah, `isActive` is true when a task is running and `false` when it has been interrupted/replaced by another task.

Reply:
Ok